javascript/src/components/dashboard/DashboardList.jsx

```diff
@@ -0,0 +1,49 @@
+const React = require('react');
+const Alert = require('react-bootstrap').Alert;
+
+const Dashboard = require('./Dashboard');
+const EditDashboardModalTrigger = require('./EditDashboardModalTrigger');
+
+const PermissionsMixin = require('../../util/PermissionsMixin');
```

Comment:
Could you please use [module imports](https://github.com/airbnb/javascript#10.1)? That way imports are theoretically independent of the module loader user.